neuralprophet/forecaster.py

```diff
@@ -406,6 +400,10 @@ def add_lagged_regressor(
             normalize : bool
                 optional, specify whether this regressor will benormalized prior to fitting.
                 if ``auto``, binary regressors will not be normalized.
+            handle_negatives : str, float, int
```

Comment:
It may make sense to structure this docstring as a list of options that shows up nicely in the docs, e.g.         [trend_reg_threshold : bool, optional](https://github.com/ourownstory/neural_prophet/blob/93b26a04c640a2589af634f115d2e6c0204e6d8e/neuralprophet/forecaster.py#L79)